bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -806,18 +671,18 @@ TEST(requestPreprocessingState_test, batchCancelledNoConsensusReached) {
         std::make_unique<PreProcessBatchReplyMsg>(clientId, senderId, repliesList, cid, overallRepliesSize, viewNum));
   }
 
-  usleep(reqTimeoutMilli * 1000);
+  usleep(reqTimeoutMilli * 10000);
```

Comment:
Is this timeout increase necessary?

Reply:
yes, this test failed on my machine, and as it doesn't test performance, it shouldn't be so sensitive.